tests/dummy/app/pods/docs/guides/main/table-customization/template.md

```diff
@@ -65,8 +65,8 @@ components:
           <h.row as |r|>
             <r.cell as |columnValue columnMeta|>
               <EmberTh::SortIndicator @columnMeta={{columnMeta}} />
-              Custom Header {{columnValue.name}}
-              <EmberTh::Resizehandle @columnMeta={{columnMeta}} />
+              {{columnValue.name}}
```

Comment:
By dropping `Custom Header` haven't you made this the same as the default header? 🤔

Reply:
I guess that's true, but this demo is to show that you can customize whether (and where) the sort-indicator and resize-handle appear. I'm not sure why it said "Custom Header" in the first place, probably copy-pasta'd here from another demo on this page. I think it is fine to remove. (fwiw, this demo was added _after_ the last docs were published (ie v 2.0.0-beta.4), so it's never been seen "publicly").

Reply:
In #708 I add some additional docs and interactivity around this demo (namely, the ability to turn on/off the sort-indicator and resize-handle), so I think when that is merged this demo will be fully comprehensible without the "Custom Header" text.